packages/salesforcedx-vscode-core/src/commands/auth/forceAuthDevHub.ts

```diff
@@ -34,22 +34,36 @@ import { isDemoMode } from '../../modes/demo-mode';
 import { isSFDXContainerMode } from '../../util';
```

Comment:
Do you know why the above package-lock file is so much bigger then it was previously? It doesn't seem valid based on the changes in this PR.

Reply:
I am looking at the diff and some of the changes are formatting changes and what looks like just more metadata about the dependancies. I looks like the `lockfileversion` went from 1 ---> 2 which might explain a lot of the changes.
<img width="844" alt="image" src="https://user-images.githubusercontent.com/25188632/167166366-39423e0c-0e57-49d0-8556-72d4589ff007.png">

I didn't need to change anything with the dependancies for this work so we should be able to remove the commit with the lockfile if you want? The reason its in there is based on [this conversation](https://salesforce-internal.slack.com/archives/G0213NY8PMH/p1651253372344739) with Mohan

Reply:
hum.. reverting seems safest here, but let's see what @mohanraj-r thinks.

Reply:
Yes please revert to version 1 - I have run into issues with getting vsce packaging to work correctly with npm v7/8 and the new lock file versions

Reply:
Done in [Revert "fix: update package lock"](https://github.com/forcedotcom/salesforcedx-vscode/pull/4077/commits/30a81d8db131129ed794ddf4ebcaa750977867ba)